test/functional/wiki_controller_test.rb

```diff
@@ -558,6 +558,16 @@ def teardown
     assert_not_nil :topics
   end
 
+  test 'should get methods page and show questions count' do
+    nodes(:method).add_tag('questions:spectrometer', users(:bob))
```

Comment:
@jywarren ...see this line : 

https://github.com/publiclab/plots2/blob/34e78b8d42a6820cd17aa5633c4b3d18b8d72fbd/app/models/node.rb#L777

undefined local variable ,  right ?